orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/AbstractWaitForClusterWideClouddriverTask.groovy

```diff
@@ -68,7 +68,8 @@ abstract class AbstractWaitForClusterWideClouddriverTask extends AbstractCloudPr
       def isMatch = it.region == deployServerGroup.region && it.name == deployServerGroup.name
       isMatch
     })
-    log.info "Server groups matching $deployServerGroup : $matchingServerGroups"
+
+    log.info("${this.getClass().getSimpleName()}: Server groups matching $deployServerGroup : $matchingServerGroups")
```

Comment:
Curious ... was the default `logger_name` wrong and is that a byproduct of using `@Slf4j` that could have been solved by explicitly doing a `LoggerFactory.getLogger(getClass())`?

It'd be nice to not litter log messages with the classes names when they're also represented in the logger context.

Reply:
yeah, it was always AbstractWaitForClusterWideClouddriverTask, let me use the `getLogger(getClass())` that's way better, thanks!